Symfony/CS/Tokenizer/Transformer/ArrayTypehint.php

```diff
@@ -28,8 +28,11 @@ public function process(Tokens $tokens)
     {
         foreach ($tokens->findGivenKind(T_ARRAY) as $index => $token) {
             $nextIndex = $tokens->getNextMeaningfulToken($index);
-            $nextToken = $tokens[$nextIndex];
+            if (null === $nextIndex) {
+                return;
```

Comment:
how it behaves after https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1541/files#r45877095 ?